app/routes/api/applications.js

```diff
@@ -18,7 +18,7 @@ module.exports = [{
     },
     handler: async (request, h) => {
       const application = (await get(request.params.ref))
```

Comment:
Not a big deal but we dont need the brackets around the await call do we? I'm surprised the linting doesnt fix this.

Reply:
No they serve no purpose. Keen to get Prettier working in tandem with 'standard' lint styling so we can have stuff like this formatted away on save.